chain/chunks/src/shards_manager_actor.rs

```diff
@@ -2091,9 +2094,10 @@ impl ShardsManagerActor {
             "type" = <&'static str>::from(&request)
         )
         .entered();
+        let me = self.validator_signer.get().map(|signer| signer.validator_id().clone());
         match request {
             ShardsManagerRequestFromClient::ProcessChunkHeaderFromBlock(chunk_header) => {
-                if let Err(e) = self.process_chunk_header_from_block(&chunk_header) {
+                if let Err(e) = self.process_chunk_header_from_block(&chunk_header, me.as_ref()) {
```

Comment:
nit: I see you call `.as_ref()` everywhere, can you just do it once where you create `me`?

Reply:
I would have to do:
```
let me = self.validator_signer.get().map(|signer| signer.validator_id().clone());
let me = me.as_ref();
```
I have not strong opinion which way is better, will change it :)

Reply:
Done https://github.com/near/nearcore/pull/11400/commits/b0535005c027328c79837e600fda95f0ac23062b